alf/trainers/evaluator.py

```diff
@@ -74,10 +74,14 @@ def __init__(self, config: TrainerConfig, conf_file: str):
                       pre_configs, num_envs, config.root_dir, seed))
             self._worker.start()
         else:
-            self._env = create_environment(
-                for_evaluation=True,
-                num_parallel_environments=num_envs,
-                seed=seed)
+            if config.shared_train_eval_env:
+                self._env = alf.get_env()
+                self._env.reset()
```

Comment:
assert async_eval = False?

Reply:
Good point. Added assertion.

Reply:
Commit not pushed to the right remote?

Reply:
ah, yes, that was what happened ... now pushed

Reply:
Happened to me also.  It's hard to remember, especially now that we don't change alf that often.  We can probably remove the other remote.